### PR TITLE
Adding Entity ChecklistLibrary, ChecklistLibraryChecklistItem, TicketChecklistLibrary and removing duplicate ShippingType from Client.php

### PR DESCRIPTION
**Pull request by srimanaravind**
Adding Entity ChecklistLibrary, ChecklistLibraryChecklistItem, TicketChecklistLibrary and removing duplicate ShippingType from Client.php

### CODE REVIEW

**Comment by srimanaravind** [comment]
@reynolek I am just adding few missing Check list classes and also removing one duplicate class entry on Client.php, could you please review and merge it please? bit urgent as I need to deploy it before end of this month, thanks a lot

File: src/Client.php [hunk 1]
```diff
@@ -143,7 +145,6 @@ class Client extends \SoapClient
         'SubscriptionPeriod'                => 'ATWS\AutotaskObjects\SubscriptionPeriod',
         'Survey'                            => 'ATWS\AutotaskObjects\Survey',
         'SurveyResults'                     => 'ATWS\AutotaskObjects\SurveyResults',
-        'ShippingType'                      => 'ATWS\AutotaskObjects\ShippingType',
```
Comment (reynolek) [inline src/Client.php:146]: Intentional removal? If so, be sure to remove the object file as well.

Comment (srimanaravind) [reply src/Client.php:146]: Hi @reynolek  Yes, intentional removal as it got declared multiple times

Comment (srimanaravind) [reply src/Client.php:146]: @reynolek check line 142, its already there

**Review by srimanaravind — COMMENTED** [review]
@reynolek @jsswift that line was removed as it was a duplicate entry, check line 142 and the entity is still there, thanks a lot

**Comment by srimanaravind** [comment]
@tabacco @tresni @jsswift will be great if someone can look at this please?

**Comment by RileyCarney** [comment]
@srimanaravind may be helpful to look over the autotask powershell module to see how it utilizes the `-all` flag, which pulls the entire data. Believe they just iterate while offsetting until the offset data = $null

**Comment by srimanaravind** [comment]
> @srimanaravind may be helpful to look over the autotask powershell module to see how it utilizes the `-all` flag, which pulls the entire data. Believe they just iterate while offsetting until the offset data = $null

@RileyCarney thanks for the reply, I can get data from autotask without these new classes, they only needed as I need to save the data into autoask

**Comment by rneufeldcisco** [comment]
Thanks!

**Pull request merged**